sdk/textanalytics/Azure.AI.TextAnalytics/tests/TextAnalyticsClientLiveTests.cs

```diff
@@ -11,18 +11,24 @@
 
 namespace Azure.AI.TextAnalytics.Tests
 {
+    [ClientTestFixture(
+        TextAnalyticsClientOptions.ServiceVersion.V3_0,
+        TextAnalyticsClientOptions.ServiceVersion.V3_1_Preview_1)]
     public class TextAnalyticsClientLiveTests : RecordedTestBase<TextAnalyticsTestEnvironment>
     {
-        public TextAnalyticsClientLiveTests(bool isAsync) : base(isAsync)
+        private readonly TextAnalyticsClientOptions.ServiceVersion _serviceVersion;
+
+        public TextAnalyticsClientLiveTests(bool isAsync, TextAnalyticsClientOptions.ServiceVersion serviceVersion) : base(isAsync)
         {
+            _serviceVersion = serviceVersion;
             Sanitizer = new TextAnalyticsRecordedTestSanitizer();
         }
 
         public TextAnalyticsClient GetClient(AzureKeyCredential credential = default, TextAnalyticsClientOptions options = default)
```

Comment:
You also probably need to check for any callers who pass `options` and make sure they're also using `_serviceVersion` when constructing an instance of `TextAnalyticsClientOptions`.

Reply:
Thank you!